tests/helpers/__init__.py

```diff
@@ -159,6 +181,44 @@ def fetch_ceph_status(juju: jubilant.Juju, app: str) -> dict[str, Any]:
     return json.loads(output)
 
 
+def wait_for_ceph_health_ok(
```

Comment:
Just a note for future us: We need to organize the helpers into kinds (jujuops helpers, microceph helpers etc) once this file size reaches 1k lines.